new-bot/src/bot/filters/is-admin.ts

```
@@ -1,4 +1,4 @@
 import { isUserHasId } from "grammy-guard";
 import { config } from "#root/config.js";
 
-export const isAdmin = isUserHasId(...config.BOT_ADMINS);
+export const isAdmin = isUserHasId(...config.BOT_ADMINS.map(x => parseInt(x)));
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)